lib/misc/cells.js

```diff
@@ -0,0 +1,64 @@
+(function() {
+  var scopes, values;
+
+  scopes = require('./scopes');
+
+  values = require('underscore-plus').values;
```

Comment:
Thanks, but couldn't get that to work, solved it by removing dependency on underscore.